<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingSetupWidget</class>
 <widget class="QWidget" name="VotingSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="cbUseThisPage">
     <property name="text">
      <string>Use this page</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainPanel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Who can vote</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QRadioButton" name="rbAllCanVote">
           <property name="text">
            <string>All can vote</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbNoOneCanVote">
           <property name="text">
            <string>No one can vote</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbSpectatorsCantVote">
           <property name="text">
            <string>Spectators can't vote</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Minimum number of players required to call a vote:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinMinimumNumberOfPlayersRequiredToCallAVote">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_16">
        <property name="title">
         <string>Allow specific votes</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="1">
          <widget class="QCheckBox" name="cbDuelLimitVote">
           <property name="text">
            <string>duellimit</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cbPointLimitVote">
           <property name="text">
            <string>pointlimit</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="cbTimeLimitVote">
           <property name="text">
            <string>timelimit</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="cbWinLimitVote">
           <property name="text">
            <string>winlimit</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cbChangeMapVote">
           <property name="text">
            <string>changemap</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="cbKickVote">
           <property name="text">
            <string>kick</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="cbMapVote">
           <property name="text">
            <string>map</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cbFragLimitVote">
           <property name="text">
            <string>fraglimit</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cbForceSpectatorVote">
           <property name="text">
            <string>force players to spectate</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbVoteFloodingProtectionEnabled">
        <property name="text">
         <string>Vote flooding protection enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>59</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbAllCanVote</tabstop>
  <tabstop>rbNoOneCanVote</tabstop>
  <tabstop>rbSpectatorsCantVote</tabstop>
  <tabstop>spinMinimumNumberOfPlayersRequiredToCallAVote</tabstop>
  <tabstop>cbKickVote</tabstop>
  <tabstop>cbFragLimitVote</tabstop>
  <tabstop>cbChangeMapVote</tabstop>
  <tabstop>cbDuelLimitVote</tabstop>
  <tabstop>cbMapVote</tabstop>
  <tabstop>cbPointLimitVote</tabstop>
  <tabstop>cbTimeLimitVote</tabstop>
  <tabstop>cbWinLimitVote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbUseThisPage</sender>
   <signal>toggled(bool)</signal>
   <receiver>mainPanel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>2</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
